index.html

```diff
@@ -16,7 +16,7 @@
 </script>
     <link rel="stylesheet" href="style1.css">
     <link rel="stylesheet" href="style2.css">
-    <title>portfolio</title>
+    <title>Homepage</title>
```

Comment:
- [ ] To make your site more accessible, your page title should adequately and briefly describes the content of the page (e.g. **Mugisha's portifolio**,  **Mugisha full-stack webdeveloper**). Please kindly fix that and update your pull request summary.